kubernetes/kong-gw/kong.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: integration-service-gw
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /api/$2
+spec:
+  ingressClassName: kong
+  rules:
+  - http:
+      paths:
+      - path: /api(/|$)(.*)/channel-info
+        pathType: Prefix
+        backend:
+          service:
+            name: audit-trail-gw
+            port:
+              number: 3002
+      - path: /api(/|$)(.*)/channels
+        pathType: Prefix
+        backend:
+          service:
+            name: audit-trail-gw
+            port:
+              number: 3002
+      - path: /api(/|$)(.*)/subscriptions
+        pathType: Prefix
+        backend:
+          service:
+            name: audit-trail-gw
+            port:
+              number: 3002
+      - path: /api(/|$)(.*)/authentication
+        pathType: Prefix
+        backend:
+          service:
+            name: ssi-bridge
+            port:
+              number: 3001
+      - path: /api(/|$)(.*)/identities
+        pathType: Prefix
+        backend:
+          service:
+            name: ssi-bridge
+            port:
+              number: 3001
+      - path: /api(/|$)(.*)/verification
+        pathType: Prefix
+        backend:
+          service:
+            name: ssi-bridge
+            port:
+              number: 3001
+---
+# TODO#415 this should later link to the docs and info of each microservice
```

Comment:
Are you doing this in another task? If task is already created, please remove the comment

Reply:
Yes task is #415 as linked in the todo just marked it as todo so we are aware this needs to be changed